eth-connector/src/lib.rs

```diff
@@ -30,33 +27,32 @@ use near_plugins::{
 use near_sdk::{
     assert_one_yocto,
     borsh::{self, BorshDeserialize, BorshSerialize},
-    collections::{LazyOption, LookupSet},
+    collections::LazyOption,
     env,
     json_types::U128,
     near_bindgen, require, AccountId, Balance, BorshStorageKey, Gas, PanicOnDefault, Promise,
     PromiseOrValue,
 };
 use serde::{Deserialize, Serialize};
-use std::collections::HashSet;
 
 pub mod connector;
 pub mod connector_impl;
 pub mod deposit_event;
 pub mod errors;
 pub mod log_entry;
 pub mod migration;
-pub mod proof;
 pub mod types;
 
 const GAS_FOR_RESOLVE_TRANSFER: Gas = Gas(5 * Gas::ONE_TERA.0);
 const GAS_FOR_FT_TRANSFER_CALL: Gas = Gas(25 * Gas::ONE_TERA.0 + GAS_FOR_RESOLVE_TRANSFER.0);
+const GAS_FINISH_WITHDRAW: Gas = Gas(5 * Gas::ONE_TERA.0);
 
 #[derive(BorshSerialize, BorshStorageKey)]
 enum StorageKey {
     FungibleToken = 0x1,
-    Proof = 0x2,
+    _Proof = 0x2,
     Metadata = 0x3,
-    EngineAccounts = 0x4,
+    _EngineAccounts = 0x4,
```

Comment:
Maybe it is better to remove the unused fields, but then we need to think how to migrate the testnet storage

Reply:
Let's better leave it as it is now to make it easier with the testnet.